src/Pages/Index.cshtml

```diff
@@ -100,21 +100,21 @@
         </div>
         <div class="page-section_row row">
             <div class="page-section_column page-section_column--narrow col-md-4 pad-fix--all text-center">
-                <img src="~/img/comp.png">
+                <img src="~/img/comp.png" alt="Projects">
                 <h3>Projects</h3>
                 <p>
                     There&apos;s a growing list of projects that are currently taking advantage of our open-source Foundation, benefitting from commercially friendly IP, copyright terms, and licenses.
                 </p>
             </div>
             <div class="page-section_column page-section_column--narrow col-md-4 pad-fix--all text-center">
-                <img src="~/img/globe_icon.png">
+                <img src="~/img/globe_icon.png" alt="Community">
                 <h3>Community</h3>
                 <p>
                     We work not only with Microsoft, but with the broader industry to increase exposure of .NET&apos;s capabilities. All our resources are designed to support developers and your projects, and help promote your endeavors with like-minded folks!
                 </p>
             </div>
             <div class="page-section_column page-section_column--narrow col-md-4 pad-fix--all text-center">
-                <img src="~/img/Ellipse_1.png">
+                <img src="~/img/Ellipse_1.png" class="Non-profit">
```

Comment:
What's this CSS class do? Did you mean to do an `alt` tag here?

Reply:
Almost certainly, will fix tonight.